ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/duplicate-vrfs-id-conflict.yml

```diff
@@ -35,6 +37,8 @@ tenants:
     mac_vrf_vni_base: 10000
     vrfs:
       - name: VRF1
+        # Making VNI the same and ID different
+        vrf_vni: 1
```

Comment:
Should be opposite. The title is ID conflict, so ID should be the same. Not VNI.

Reply:
nvm. my bad.